email.go

```diff
@@ -375,6 +371,15 @@ func (e *Email) categorizeAttachments() (htmlRelated, others []*Attachment) {
 	return
 }
 
+// streamToBytes is a helper for compatibility old and new logic
+func streamToBytes(r io.Reader) (b []byte, err error) {
+	var buffer bytes.Buffer
+	if _, err = io.Copy(&buffer, r); err != nil {
+		return
+	}
+	return buffer.Bytes(), nil
```

Comment:
This function does the same as `io.ReadAll`. So I suggest removing this function and using the one from the io package or changing this function to:

```suggestion
	return io.ReadAll(r)
```